{"uuid":"025c86c2-608e-41fb-b789-3a2e68366f9b","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["cc104053-73b3-4d9d-be62-70f81c85fe00"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696723353404,"stop":1696723353404},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696723353470,"stop":1696723353470},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696723353559,"stop":1696723353560},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696723355595,"stop":1696723355595}],"start":1696723353207,"stop":1696723355595}